Samples/Build.Common.Cpp.props

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- This file is used to set configuration option for all projects. -->
```

Comment:
>for all projects. [](http://example.com/codeflow?start=53&length=17)

This part doesn't make sense to me, TBH. The file will set it wherever it is imported, right?